src/redux/data.ts

```diff
@@ -54,7 +67,7 @@ import {
 import WalletTypes from '@rainbow-me/helpers/walletTypes';
 import { Navigation } from '@rainbow-me/navigation';
 import { triggerOnSwipeLayout } from '@rainbow-me/navigation/onNavigationStateChange';
-import networkTypes from '@rainbow-me/networkTypes';
+import networkTypes, { Network } from '@rainbow-me/networkTypes';
```

Comment:
can remove the use of `networkTypes` and just replace with `Network`

Reply:
@jinchung Definitely, I updated it to only use `Network`. Thanks!